docs/deployment/setup.md

```diff
@@ -0,0 +1,117 @@
+# KV-Cache Manager Setup Guide
+
+This guide provides a complete walkthrough for setting up and testing the example llm-d-kv-cache-manager system. You will deploy a vLLM with LMCache and Redis, then run an example application that demonstrates KV cache indexing capabilities.
+
+By following this guide, you will:
+
+1. **Deploy the Infrastructure**: Use Helm to set up a Kubernetes cluster running:
```

Comment:
Technically the Kubernetes cluster is a pre-requisite and is not being set up here. This is well reflected below, perhaps the postfix sentence can just be dropped.